src/gui/modals/SetupDialog.cpp

```diff
@@ -212,7 +212,7 @@ SetupDialog::SetupDialog(ConfigTab tab_to_open) :
 	auto generalControlsLayout = new QVBoxLayout;
 	generalControlsLayout->setSpacing(10);
 
-	auto addLedCheckBox = [&XDelta, &YDelta, this](const QString& ledText, TabWidget* tw, int& counter,
+	auto addLedCheckBox = [&](const QString& ledText, TabWidget* tw, int& counter,
```

Comment:
Is getting rid of the `this` capture is right? The `SLOT(showRestartWarning())` is for the `SetupDialog`s function.

Reply:
A `[&]` lambda capture is identical to a `[&, this]` lambda capture, so this is fine.